src/main.ts

```diff
@@ -32,6 +39,9 @@ async function run(): Promise<void> {
     // Get the minimum mask length.
     const minMaskLength = parseInt(getInput('min_mask_length'));
 
+    // Get the setting for whether to export secrets as environment variables.
+    const setEnv = getBooleanInput('set_env');
```

Comment:
We prefer to use `parseBoolean` from google-github-actions/actions-utils; `getBooleanInput` has some pretty nasty edge cases.